keras_nlp/src/models/resnet/resnet_presets.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ResNet preset configurations."""
+
+backbone_presets = {
+    "resnet_18": {
```

Comment:
should we add v1 in the preset name?

Reply:
We could. No strong opinion. Most models on HF forgo this, and only add it for v2.

Compare...
https://huggingface.co/models?sort=trending&search=resnet+v1
https://huggingface.co/models?sort=trending&search=resnet+v2